### DISCUSSION TITLE
Камеры со встроенной SD картой

### DISCUSSION THREAD

**Discussion by anisart** [Arkady23/V380-Web#51] [General]
Правильно понимаю, что кроме как физически записать на SD карту, установить Web сервер не получится? 
Имею на руках дверной глазок со встроенной картой. Попробую конечно его вскрыть ещё, может память не распаяна.

Прошивка Hwv380E22_WF3 MINLQ7_20201023

**Comment by anisart** [comment]
Да, глазок разбирается, откручивается от трубки (резьба проклеена). Карта достается, а дальше как обычно.

![PXL_20221110_082200986](https://user-images.githubusercontent.com/693481/201047020-59319ec9-744e-4d2b-a759-bfebd802cb28.jpg)
![PXL_20221110_082303933](https://user-images.githubusercontent.com/693481/201047043-25acf043-83c5-416a-bd0b-7b22e24be6c2.jpg)

**Reply by Arkady23** [reply]
По идее так. Разве, что вы смогли бы запустить telnet, то можно было бы не разбирать.

**Reply by Anton-Ka** [reply]
> Да, глазок разбирается, откручивается от трубки (резьба проклеена). Карта достается, а дальше как обычно.
> 
> ![PXL_20221110_082200986](https://user-images.githubusercontent.com/693481/201047020-59319ec9-744e-4d2b-a759-bfebd802cb28.jpg) ![PXL_20221110_082303933](https://user-images.githubusercontent.com/693481/201047043-25acf043-83c5-416a-bd0b-7b22e24be6c2.jpg)

Отличная новость! Как раз думал, как безопасно подрезать трубку под размер двери.
Подскажите, как сильно приклеена резьба? Большое ли усилие нужно приложить, чтобы стронуть?

**Reply by anisart** [reply]
Я просто ножем расковырял все наплывы клея, а дальше уже небольшое усилие. Думаю, что можно ещё и плату с разъёмом подрезать с подпайкой питания. Буду себе так делать

**Reply by Anton-Ka** [reply]
Понял.
Спасибо большое за ответ!
Буду пробовать.
